bokehjs/test/integration/glyphs/webgl.ts

```diff
@@ -58,4 +59,18 @@ describe("webgl", () => {
 
     await display(row([p0, p1]))
   })
+
+  it("should support updating line", async () => {
+    // Issue #8346.
```

Comment:
Would have been better to add this regression test under regression tests, then issue number would be a natural part of test description.

Reply:
OK, I have moved it to `regressions.ts`